squarelet/organizations/views.py

```diff
@@ -298,18 +299,47 @@ def post(self, request, *args, **kwargs):
     def _handle_add_member(self, request):
         addmember_form = AddMemberForm(request.POST)
         if not addmember_form.is_valid():
-            messages.error(request, addmember_form.errors["emails"][0])
+            messages.error(
+                request, addmember_form.errors.get("emails", ["Invalid input."])[0]
+            )
         else:
             emails = addmember_form.cleaned_data["emails"]
 
-            # Create an invitation and send it to the given email addresses
             for email in emails:
+                is_already_member = Membership.objects.filter(
```

Comment:
I feel like this new functionality would be better as a method on the Org model that can be called from the view - try to keep the views lean